src/utils/performance.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export type HighResolutionTime = [number, number];
+
+export function calculateElapsedTime(startProcessingTime: HighResolutionTime): number {
```

Comment:
Handling nanoseconds is overkill. We did that before.

For CDP because it's a lot of information, we are going to drop the nanoseconds.
For DAP because it's not that much information, and it's already there, we are leaving it as-is.